tests/unit_tests/test_weakattr.py

```diff
@@ -0,0 +1,41 @@
+import mock
+import pytest
+
+from server.weakattr import WeakAttribute
+
+
+@pytest.fixture(params=("typed", "untyped"))
+def obj(request):
+    """An object with a WeakAttribute"""
+    if request.param == "untyped":
+        class Foo(object):
```

Comment:
Every class inherits from `object` implicitly, this is just making it explicit I guess. BTW metaclasses inherit from `type` instead of `object`, that's what makes them different.